docs/stream/event_types.mdx

```diff
@@ -0,0 +1,160 @@
+
+<CodeTabs>
+<TabItem value="js">
+
+```js
+import { Svix } from "svix";
+
+const streamEventTypeOut = await svix.stream.eventType.create({
+    name: "user.signup",
+    description: null,
+    featureFlags: ["cool-new-feature"]
+});
+```
+
+</TabItem>
+<TabItem value="py">
+
+```python
+from svix.api import Svix, StreamEventTypeIn
+
+stream_event_type_out = svix.stream.event_type.create(StreamEventTypeIn(
+    name="user.signup",
+    description=None,
+    feature_flags=["cool-new-feature"]
+))
+```
+
+</TabItem>
+<TabItem value="go">
+
+```go
+import (
+	svix "github.com/svix/svix-webhooks/go"
+)
+
+svixClient := svix.New("AUTH_TOKEN", nil)
+streamEventTypeOut, err := svixClient.Stream.EventType.Create(ctx, &StreamEventTypeIn{
+    Name: "user.signup",
+    Description: nil,
+    FeatureFlags: []string{"cool-new-feature"},
+})
+```
+
+</TabItem>
+<TabItem value="rust">
+
+```rust
+let svix = Svix::new("AUTH_TOKEN".to_string(), None);
+let stream_event_type_out = svix
+    .stream()
+    .event_type()
+    .create(StreamEventTypeIn {
+        name: "user.signup".to_string(),
+        description: None,
+        feature_flags: Some(vec!["cool-new-feature".to_string()]),
+    }, None)
+    .await?;
+```
+
+</TabItem>
+<TabItem value="java">
+
+```java
+import com.svix.Svix;
+import com.svix.models.StreamEventTypeIn;
+
+
+StreamEventTypeOut streamEventTypeOut = svix.getStream().getEventType().create(new StreamEventTypeIn()
+    .name("user.signup")
+    .description(null)
+    .featureFlags(new String[]{"cool-new-feature"})
+);
+```
+
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+import com.svix.kotlin.Svix;
+import com.svix.kotlin.models.StreamEventTypeIn;
+
+val svix = Svix("AUTH_TOKEN");
+val streamEventTypeOut = svix.stream.eventType.create(StreamEventTypeIn()
+    .name("user.signup")
+    .description(null)
+    .featureFlags(arrayOf("cool-new-feature"))
+)
+```
+
+</TabItem>
+<TabItem value="ruby">
+
+```ruby
+stream_event_type_out = svix.stream.event_type.create(Svix::StreamEventTypeIn.new({
+    "name": "user.signup",
+    "description": nil,
+    "feature_flags": ["cool-new-feature"]
+}))
+```
+
+</TabItem>
+<TabItem value="csharp">
+
+```csharp
+var svix = new SvixClient("AUTH_TOKEN", new SvixOptions("https://api.us.svix.com"));
+var streamEventTypeOut = await svix.Stream.EventType.CreateAsync(new StreamEventTypeIn{
+    name: "user.signup",
+    description: null,
+    featureFlags: new string[] {"cool-new-feature"}
+});
+```
+
+</TabItem>
+<TabItem value="curl">
+
+```shell
+export SVIX_AUTH_TOKEN='AUTH_TOKEN'
+
+curl -X 'POST' \
+  'https://api.us.svix.com/api/v1/stream/event-type' \
+  -H 'Authorization: Bearer AUTH_TOKEN' \
+  -H 'Accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+        "name": "user.signup",
+        "description": null,
+        "featureFlags": ["cool-new-feature"]
+    }'
+```
+
+</TabItem>
+</CodeTabs>
+
+## Event Type Filtering
```

Comment:
This is duplicate, should link to the sinks page.

Reply:
It is a dupe, though it's a very short segment, and I actually think the docs are easier to grok w/both the specific section on event-types, and the brief description in the Sink page.